libs/blocks/comparison-table/comparison-table.css

```diff
@@ -0,0 +1,490 @@
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: var(--spacing-s);
+  border-start-start-radius: var(--comparison-border-radius);
+  border-start-end-radius: var(--comparison-border-radius);
+  font-family: var(--body-font-family);
+}
+
+.comparison-table .table-container .table-column-header button:hover {
+  cursor: pointer;
+}
+
+.comparison-table .table-container .table-column-header button .toggle-icon {
+  content: url('./img/black-bg-minus.svg');
+  max-width: 24px;
+}
+
+.comparison-table .table-container .table-column-header button[aria-expanded="false"] .toggle-icon {
+  content: url('./img/plus-sign.svg');
+}
+
+.comparison-table .table-container:has(button[aria-expanded="false"]) {
+  border-end-start-radius: var(--comparison-border-radius);
+  border-end-end-radius: var(--comparison-border-radius);
+  border-start-start-radius: var(--comparison-border-radius);
+  border-start-end-radius: var(--comparison-border-radius);
+}
+
+.comparison-table .table-container button[aria-expanded="false"] {
+  border-end-start-radius: var(--comparison-border-radius);
+  border-end-end-radius: var(--comparison-border-radius);
+}
+
+.comparison-table .table-container .table-row {
+  margin: 0 var(--spacing-s);
+  display: grid;
+  grid-template-columns: minmax(268px, 1.15fr) repeat(auto-fit, minmax(100px, 1fr));
+  gap: 12px;
+}
+
+.comparison-table[data-child-count="3"] .table-row {
+  grid-template-columns: minmax(268px, 1fr) repeat(auto-fit, minmax(100px, 1fr));
+}
+
+.comparison-table .table-container .table-row:not(:last-child) {
+  border-bottom: 1px solid var(--comparison-row-border-color);
+}
+
+.comparison-table .table-container .table-row-header {
+  text-align: start;
+  font-size: var(--type-body-xs-size);
+  font-style: normal;
+  font-weight: var(--type-detail-all-weight);
+  line-height: 130%;
+  padding: var(--spacing-s) var(--spacing-s) var(--spacing-s) 0;
+  display: flex;
+  align-items: center;
+  justify-content: flex-start;
+  margin-inline-end: 12px;
+}
+
+.comparison-table .table-container .table-row-header a {
+  display: contents;
+}
+
+.comparison-table span[data-tooltip] .icon-milo:hover {
+  cursor: pointer;
+}
+
+.comparison-table span[data-tooltip] .icon-milo:hover path {
+  color: var(--hover-border-color);
+}
+
+.comparison-table .table-container .primary-cell .icon-checkmark path {
+  color: var(--primary-cell-path-color);
+}
+
+.comparison-table .table-container .table-row .table-cell:nth-child(odd) div {
+  background-color: var(--comparison-column-background-color);
+}
+
+.comparison-table .table-container .table-cell {
+  padding: 36.5px 0;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.comparison-table .table-container .table-cell p {
+  text-align: center;
+  min-height: 18px;
+  width: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  word-break: break-word;
```

Comment:
Semi-related: Japanese word wrap logic would likely need to apply in comparison tables as well, has that been checked out yet?